### PR TITLE
RK-10731 - Update supported node versions

### PR DESCRIPTION
**Pull request by lirook**

### CODE REVIEW

**Comment by lirook** [comment]
@ElDuderinos what do you think?

File: docs/node-setup.md [hunk 1]
```diff
@@ -137,12 +137,25 @@ The NodeJS SDK supports detecting the existing source code commit in the followi
...
+Rookout supports the following NodeJS versions:
 
-| Supprt             | Versions                       |
-| ---                | ---                            |
-| **Full**           | 12 (>= 12.9), 14               |
-| **Limited**        | 8, 10, 11, 12 (< 12.9), 13, 15 |
+| Release  | Versions           |
+| ---      | ---                |
+| **12**   | 12.5.0 --> 12.22.6 |
+| **14**   | 14.0.0 --> 14.17.5 |
```
Comment (ElDuderinos) [inline docs/node-setup.md:145]: We also added 14.17.6

File: docs/node-setup.md [hunk 2]
```diff
@@ -137,12 +137,25 @@ The NodeJS SDK supports detecting the existing source code commit in the followi
...
+We strongly recommend using one of the supported NodeJS versions, however there is limited support for the following versions:
+
+| Release  | Versions           |
+| ---      | ---                |
+| **8**    | 8.0.0  --> 8.17.0  |
+| **14**   | 9.0.0  --> 9.11.2  |
+| **12**   | 10.0.0 --> 10.24.1 |
+| **14**   | 11.0.0 --> 11.15.0 |
+| **14**   | 12.0.0 --> 12.4.0  |
+| **14**   | 13.0.0 --> 13.14.0 |
```
Comment (ElDuderinos) [inline docs/node-setup.md:156]: There's some mess here with the release part.
Also, why do you need to detail the minor/patch versions? (apart for <=12.4)

**Review by ElDuderinos — CHANGES_REQUESTED** [review]

File: docs/node-setup.md (hunk 2 above)
Comment (lirook) [reply docs/node-setup.md:156]: Yep, fixing the mess.
As for the patch versions, I did it for consistency

**Review by ElDuderinos — DISMISSED** [review]

**Review by ElDuderinos — APPROVED** [review]

**Pull request merged**